pkg/kwrapper/k8s/k3s_linux.go

```diff
@@ -41,15 +34,17 @@ func getEmbedded(ctx context.Context) (bool, clientcmd.ClientConfig, error) {
 	return true, clientConfig, nil
 }
 
-func k3sServer(ctx context.Context, endpoints []string) (string, error) {
+func k3sServer(ctx context.Context) (string, error) {
+	// --cluster-init will start k3s with embedded etcd
```

Comment:
nit: I think we can remove this comment since this PR and attached issue will indicate this as such. Perhaps, this would be better for a PR comment or a link to the k3s docs.

Reply:
+1 on this nit. Will approve after it's resolved.